sha3/src/main/java/com/theromus/sha/Keccak.java

```diff
@@ -62,23 +192,22 @@
 
         // Squeezing phase
         ByteArrayOutputStream byteResults = new ByteArrayOutputStream();
-        int tOutputLen = parameter.getOutputLen() / 8;
+        int tOutputLen = (int) Math.ceil(customLen / 8.0);
+        byteResults.write(0);
         while (tOutputLen > 0) {
             blockSize = min(tOutputLen, rateInBytes);
-            for (int i = 0; i < blockSize; i++) {
```

Comment:
Bring back curly braces. IMAO, this make parts of code more readable :)

Reply:
I don't fully agree, but okay.